### PR TITLE
update python snippets

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: docs/howtos/DataPlane Generation - DPG/04renaming.mdx [hunk 1]
```diff
@@ -136,9 +135,10 @@ using Azure.ClientGenerator.Core;
 <TabItem value="python" label="Python">
 
 ```python
-class OutputOptions:
-  output_name: str = rest_field(rest_name="name", readonly=True)
-FIXME
+class InputModel:
```
Comment (tadelesh) [inline docs/howtos/DataPlane Generation - DPG/04renaming.mdx:138]:
```suggestion
class InputOptions:
```

File: docs/howtos/DataPlane Generation - DPG/04renaming.mdx [hunk 2]
```diff
@@ -136,9 +135,10 @@ using Azure.ClientGenerator.Core;
...
+  input_name: str = rest_field(rest_name="name", readonly=True)
+
+response: InputModel = client.get_computed_model(name="name")
```
Comment (tadelesh) [inline docs/howtos/DataPlane Generation - DPG/04renaming.mdx:141]:
```suggestion
response: InputOptions = client.get_computed_model(input_name="name")
```

File: docs/howtos/DataPlane Generation - DPG/04renaming.mdx [hunk 3]
```diff
@@ -40,11 +40,10 @@ op GetModel is ResourceRead<InputModel>;
 <TabItem value="python" label="Python">
 
 ```python
-class OutputModel:
+class InputModel:
   name: str = rest_field(readonly=True)
 
-response: OutputModel = client.get(name="name")
-FIXME
+response: InputModel = client.get(name="name")
```
Comment (tadelesh) [inline docs/howtos/DataPlane Generation - DPG/04renaming.mdx:46]:
```suggestion
response: InputModel = client.get_model(name="name")
```

File: docs/howtos/DataPlane Generation - DPG/04renaming.mdx [hunk 4]
```diff
@@ -212,7 +212,10 @@ using Azure.ClientGenerator.Core;
 <TabItem value="python" label="Python">
 
 ```python
-client.get_computed_model()
+class InputModel:
```
Comment (tadelesh) [inline docs/howtos/DataPlane Generation - DPG/04renaming.mdx:215]:
```suggestion
class ReadModel:
```

File: docs/howtos/DataPlane Generation - DPG/04renaming.mdx [hunk 5]
```diff
@@ -212,7 +212,10 @@ using Azure.ClientGenerator.Core;
...
+  input_name: str = rest_field(rest_name="name", readonly=True)
+
+client.get_computed_model(name="name")
```
Comment (tadelesh) [inline docs/howtos/DataPlane Generation - DPG/04renaming.mdx:218]:
```suggestion
response: ReadModel = client.get_computed_model(name="name")
```

File: docs/howtos/DataPlane Generation - DPG/04renaming.mdx (hunk 4 above)
Comment (xiangyan99) [reply docs/howtos/DataPlane Generation - DPG/04renaming.mdx:215]: Get model is the method name, not the model name, why we rename the mode?

Comment (tadelesh) [reply docs/howtos/DataPlane Generation - DPG/04renaming.mdx:215]: oh, my fault.

File: docs/howtos/DataPlane Generation - DPG/04renaming.mdx (hunk 5 above)
Comment (xiangyan99) [reply docs/howtos/DataPlane Generation - DPG/04renaming.mdx:218]: And same here, right?

**Review by tadelesh — APPROVED** [review]

**Pull request merged**